### ISSUE TITLE
incompatible with latest blender 2.78

### ISSUE THREAD

**Issue by camic** [waylow/Add_Camera_Rigs#5]
cannot activate:
attribute error RNA_types has no attributes  line 915

**Comment by waylow** [comment]
Does this help?
https://github.com/waylow/Add_Camera_Rigs/issues/4

**Comment by thelapisfox** [comment]
Installation instructions are not very intuitive, but what I had to do was install the entire folder of scripts for the addon directly into AppData\Roaming\Blender Foundation\Blender\2.78\scripts\addons. Once I did that and started blender back up, it worked fine.


Edit: Hmmm. After double checking some stuff, it seems like I might have made a mistake with my own report. I do not have it installed and when trying to install it (in any way at all), I get the same error as you now. And, looking at my script folder, I think my comment from when I closed it, was not for this addon at all, but for a different addon. Some how I posted my reply onto the wrong bug report.....
